packages/components/atoms/f-card/babel.config.js

```diff
@@ -10,13 +10,11 @@ module.exports = api => {
     if (!isTest) {
         api.cache(true); // Caches the computed babel config function – https://babeljs.io/docs/en/config-files#apicache
         presets.push(['@vue/app', { useBuiltIns: builtIns }]);
+    } else {
+        // use current node version for transpiling test files
+        presets.push(['@babel/env', { targets: { node: 'current' } }]);
```

Comment:
This code is present in all other components, and was preventing ES6 syntax from being used in the tests.